tests/devapps/WAM/NetCoreWinFormsWam/Form1.cs

```diff
@@ -329,7 +329,7 @@ private async void atiBtn_Click(object sender, EventArgs e)
             }
         }
 
-        private async Task<AuthenticationResult> RunAtiAsync(IPublicClientApplication pca)
+        private async Task<AuthenticationResult> RunAtiAsync(IPublicClientApplication pca, CancellationToken cancellationToken)
```

Comment:
nit: instead of passing the cancellation token to ATS and ATI methods, just create it right before ExecuteAsync?